src/lib/theme/index.tsx

```diff
@@ -26,6 +26,7 @@ export const lightTheme: Colors = {
   dialog: '#FFFFFF',
 
   // text
+  accentText: 'hsl(0, 0%, 100%)',
```

Comment:
use #FFFFFF to be consistent with other definitions (and how it comes from figma)

Reply:
We *should* be using HSL everywhere. Hex is impossible to parse at a glance but HSL has human-readable info. Figma output is user-configurable, so you can read out hsl values wherever you would hex.

Reply:
can you use #FFFFFF to be consistent here, and update _all_ values to use HSL in another PR?